go-controller/pkg/clustermanager/userdefinednetwork/template/net-attach-def-template.go

```diff
@@ -203,6 +257,22 @@ func validateIPAM(ipam *userdefinednetworkv1.IPAMConfig) error {
 	return nil
 }
 
+func validateVLANConfig(vlanConfig *userdefinednetworkv1.VLANConfig) error {
+	if vlanConfig == nil {
+		return nil
+	}
+	if vlanConfig.Mode != userdefinednetworkv1.VLANModeAccess {
+		return fmt.Errorf("vlan.mode support 'Access' mode only")
+	}
+	if vlanConfig.Access == nil {
+		return fmt.Errorf("when vlan.mode is 'Access', vlan.access is required")
+	}
+	if vlanConfig.Access.ID <= 0 || vlanConfig.Access.ID >= 4095 {
```

Comment:
CEL validation on this one?

Reply:
CEL validation for VLAN ID https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5010/files#diff-073a0bb9fa1b183caf787b824a1bddd130ec004579ff07082596b7426d4c9b8cR192-R198

Please see https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5010/files/082457cd9d6bf3065da210aff8ea2f643b0b4af2..5162882165cb2379c9f36d9a20cac5f679380e47#r2046484060